lib/cli.js

```diff
@@ -8,7 +8,10 @@ var humanizeDuration = require("humanize-duration");
 var nopter = require("nopter");
 var spinner = require("char-spinner");
 
-
+process.on('unhandledRejection', (reason, p) => {
+	console.error("Unhandled Rejection: " + reason);
+	process.exit(1);
```

Comment:
Doesn't this occur automatically? Also, I haven't yet dropped support for Node <= 4, so we would have to do without the arrow function.